src/types/Submissions.d.ts

```diff
@@ -18,7 +18,27 @@ type DataSubmission = {
   updatedAt: string; // # ISO 8601 date time format with UTC or offset e.g., 2023-05-01T09:23:30Z
 };
 
-type DataSubmissionStatus = "New" | "In Progress" | "Submitted" | "Released" | "Withdrawn" | "Rejected" | "Completed" | "Archived" | "Canceled";
+type SubmissionHistoryEvent = {
+  status: SubmissionStatus; // # [New, In Progress, Submitted, In Review, Approved, Rejected]
+  reviewComment: string; // # if applicable
+  dateTime: string; // # YYYY-MM-DDTHH:MM:SS format
+  userID: string;
+};
+
+type SubmissionStatus = "New" | "In Progress" | "Submitted" | "Released" | "Withdrawn" | "Rejected" | "Completed" | "Archived" | "Canceled";
+
+type HistoryEvent = {
+  status: ApplicationStatus;
+  reviewComment?: string;
+  dateTime: string; // YYYY-MM-DDTHH:MM:SSZ format
+  userID: number;
+};
+
+type Applicant = {
```

Comment:
This already exists in src/types/Application.d.ts does this have a use for data submissions?